src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml

```diff
@@ -126,7 +118,7 @@
             }
         }
 
-        if (Model.AllowEditResult)
+        if (Model.AllowedOperations.IsAllowed(AuthOperations.Games.Edit))
```

Comment:
We have to use AllowEdit property here as only 1 round games can be edited. Other games are edited automatically.  As part of this issue we need to change logic around how this value is set.